frontend/containers/Admin/UsersDetail.js

```diff
@@ -5,42 +5,41 @@ import { bindActionCreators } from 'redux'
 import { fetchUsers, promoteUser, deleteUser } from '../../actions/'
 
 import { Segment, Container, Button, Icon, Modal } from 'semantic-ui-react'
-import { withRouter, Link} from 'react-router-dom'
+import { withRouter, Link } from 'react-router-dom'
 
 import { ContactCard, DemographicsCard, EmergencyContactCard, ErrorComponent, LoadingIcon, PageTitle, PastEventsCard } from '../../components'
 
 class AdminUsersDetail extends Component {
   constructor (props) {
     super(props)
-    const { updateUserStore, users, match } = this.props
+    // I feel like this doesn't belong in constructor!!!!
```

Comment:
Probably better than retrieving each one individually in the actions below! It makes it clear that all of these are coming from the props.